pages/linux/nft.md

```diff
@@ -0,0 +1,32 @@
+# nftables
```

Comment:
There appears to be a mismatch between the name of the page and the command name. Perhaps we could change this to `nft`, and then include it's full name in the description somehow? @agnivade, what do you think?

Reply:
Right. If the command name is `nft`, that should be the title.